.travis.yml

```diff
@@ -7,18 +7,15 @@ notifications:
   email: false
 node_js:
   - 'stable'
-  - '5'
```

Comment:
any reason to remove these ones? I suppose given that gulp is quite an outdated tool now, people would be happy to have older versions supported for as long as we can?

Reply:
semantic-release requires node 8+ to work (even the version 7 used before was breaking on Travis)

Reply:
anyway, semantic-release breaks on Travis, because it does not have the required authentication (both in master and in this PR, although with different messages). So this might require more work

Reply:
yes, I think this was because of npm's big secret reset a few months back. I will have a look.